packages/react-link/src/components/Link/useLinkState.ts

```diff
@@ -3,11 +3,12 @@ import { getCode, EnterKey, SpacebarKey } from '@fluentui/keyboard-key';
 import type { LinkState } from './Link.types';
 
 /**
- * The useLink hook processes the Link draft state.
+ * The useLink hook processes the Link draft state.root.
  * @param state - Link draft state to mutate.
  */
 export const useLinkState = (state: LinkState): LinkState => {
-  const { as, disabled, disabledFocusable, href, onClick, onKeyDown: onKeyDownCallback, type } = state;
+  const { disabled, disabledFocusable, href, onClick, onKeyDown: onKeyDownCallback, type } = state;
```

Comment:
```suggestion
  const { disabled, disabledFocusable, href, onClick, onKeyDown: onKeyDownCallback, type } = state.root;
```

I can be wrong, but all (_or almost all_) should be used from `state.root` (see a previous comment about spreading `props` on `state`)